### PR TITLE
Bug 1128553 - Use the smtp.mozilla.com host for sending e-mails

### PR DESCRIPTION
**Pull request by bebef1987**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[bebef1987 Florin Strugariu (bebef1987)](https://github.com/bebef1987) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5a48e9d36497)

**Comment by try-server-hook** [comment]
[bebef1987 Florin Strugariu (bebef1987)](https://github.com/bebef1987) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=56c77423e343)

File: tests/python/gaia-ui-tests/gaiatest/utils/email/email_util.py [hunk 1]
```diff
@@ -28,8 +28,9 @@ def send(self, senders_account, mock_email):
                                        str(self.senders_account['smtp_port']))
```
Comment (davehunt) [inline tests/python/gaia-ui-tests/gaiatest/utils/email/email_util.py]: The changes in webqa-credentials will cause this to expect 'hostname' and 'port'. Please update the docstring and these occurrences.

**Comment by try-server-hook** [comment]
[bebef1987 Florin Strugariu (bebef1987)](https://github.com/bebef1987) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b46c0b941d73)

**Comment by try-server-hook** [comment]
[bebef1987 Florin Strugariu (bebef1987)](https://github.com/bebef1987) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=eac06c198905)

**Comment by davehunt** [comment]
Merged in https://github.com/mozilla-b2g/gaia/commit/4c0805f9b7a9d5fdcf7dee2658d98e0b1f16033e

**Pull request closed without merging**